babel.config.js

```diff
@@ -2,7 +2,10 @@ module.exports = function( api ) {
 	api.cache( true );
 
 	return {
-		presets: [ '@wordpress/babel-preset-default' ],
+		presets: [
+			'@wordpress/babel-preset-default',
+			'@babel/preset-env',
```

Comment:
This seems redundant, is it required?

Reply:
the preset is required for jest tests to work